WalletWasabi/Wallets/Wallet.cs

```diff
@@ -71,6 +72,12 @@ public class Wallet : BackgroundService, IWallet
 
 	public event EventHandler<WalletState>? StateChanged;
 
+	public int Id { get; }
+
+	public static int NextId => WalletCount + 1;
```

Comment:
Please explain the need of this

Reply:
Inside **WalletRepository**, when creating, importing, restoring or connecting to hardware wallet, an instance of WalletSettingsModel is created. It needs the Id of a wallet hasn't been created yet. So, the only way to know the Id in advance is to set it to [wallet count]+1. I might be wrong, though!

Reply:
This has been moved to **WalletRepository**.

WalletRepository.cs: 124

`	private static int NextWalletId() => Services.WalletManager.GetWallets().Count() + 1;`